test/tests/ver-1e/ver-1e.i

```diff
@@ -75,7 +68,9 @@
   petsc_options_iname = '-pc_type -pc_hypre_type'
   petsc_options_value = 'hypre boomeramg'
   scheme = 'crank-nicolson'
-  nl_rel_tol = 1e-9
+  nl_rel_tol = 1e-50
```

Comment:
This might be the tightest relative tolerance I have ever seen 😮

Reply:
Haha it just ensures that the `nl_abs_tol` will be used

Reply:
Ah right - just like we do for `abs` default. I would suggest that a comment is entered here for the novice MOOSE/TMAP8 user who might look at this test as an example and wonder why we're doing it.